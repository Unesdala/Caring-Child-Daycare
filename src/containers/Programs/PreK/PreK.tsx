import React, { Component } from 'react';
import CommonUtils from '../../../lib/commonUtils';
import DefaultPreKContent from './PreKContent';

export class PreK extends Component {
  public commonUtils: typeof CommonUtils = CommonUtils;

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Private Pre-K', window.screen.width); }

  render(): JSX.Element {
    return (
      <>
        <div className="page-content">
          <DefaultPreKContent />
        </div>
      </>
    );
  }
}

export default PreK;
