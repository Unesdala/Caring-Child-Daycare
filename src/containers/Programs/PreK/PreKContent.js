import React from 'react';

const PreKContent = () => (
  <main className="pre-k">
    <p />
  </main>
);

export default PreKContent;
