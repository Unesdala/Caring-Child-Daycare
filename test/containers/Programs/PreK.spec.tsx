/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { PreK } from 'src/containers/Programs/PreK/PreK';
import PreKContent from 'src/containers/Programs/PreK/PreKContent';

describe('PreK', () => {
  const wrapper = shallow(<PreK />);
  it('renders', () => {
    expect(wrapper.find(PreKContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(PreKContent).dive().find('main.pre-k-main--content').exists()).toBe(true);
  });
  
});
