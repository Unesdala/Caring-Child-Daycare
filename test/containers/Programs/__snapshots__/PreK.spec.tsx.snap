// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreK renders 1`] = `ShallowWrapper {}`;
